File: 프로그래머스/0/181842. 부분 문자열/부분 문자열.java
class Solution {
    public int solution(String str1, String str2) {
        int answer = 0;
        
        for(int i=0; i <= str2.length() - str1.length(); i++){
            boolean find = true;
            for(int j=0; j < str1.length(); j++){
                if(str2.charAt(i+j) != str1.charAt(j)){
                    find = false;
                    
                }
            }
            if(find) return 1;
        }
        return 0;
    }
}

File: 프로그래머스/0/181842. 부분 문자열/부분 문자열.kt
class Solution {
    fun solution(str1: String, str2: String): Int {
        var answer = 0
        for(i in 0 .. str2.length - str1.length){
        if(str2.slice(i until str1.length + i) == str1){
            answer = 1
            break
        }
    }
        return answer
    }
}